text/2094-nll.md

```diff
@@ -1563,6 +1563,8 @@ follows:
 
 - any loans whose region does not include P are killed;
 - if this is a borrow statement, the corresponding loan is generated;
+  - if the borrow contains a union field, loans are generated for all
```

Comment:
This is vague ("if a borrow contains a union field"), and doesn't look like the right place to specify borrows of union fields.